USAGE.md

```diff
@@ -38,6 +38,8 @@ ssl_certs_file =
 timeout = 30
 hostname = machinename
 log_file =
+import_cfg = /path/to/another/wakatime.cfg
+```
```

Comment:
A typo?

Reply:
This setting wasn't documented so I added in this PR.

Reply:
The "```" shouldn't be there

Reply:
Oops!